packages/remote-logs/src/logger.ts

```diff
@@ -129,32 +129,34 @@ export class RemoteLogger extends BifoldLogger {
     }, expirationInMinutes * 60000)
   }
 
-  public test(message: string, data?: object | undefined): void {
-    this._log?.test({ message, data })
+  public test(...msgs: unknown[]): void {
+    this._log?.test(...msgs)
   }
 
-  public trace(message: string, data?: object | undefined): void {
-    this._log?.trace({ message, data })
+  public trace(...msgs: unknown[]): void {
+    this._log?.trace(...msgs)
   }
 
-  public debug(message: string, data?: object | undefined): void {
-    this._log?.debug({ message, data })
+  public debug(...msgs: unknown[]): void {
+    this._log?.debug(...msgs)
   }
 
-  public info(message: string, data?: object | undefined): void {
-    this._log?.info({ message, data })
+  public info(...msgs: unknown[]): void {
+    this._log?.info(...msgs)
   }
 
-  public warn(message: string, data?: object | undefined): void {
-    this._log?.warn({ message, data })
+  public warn(...msgs: unknown[]): void {
```

Comment:
suggestion: I would also check to see if the log output looks the same from the original version to your updated one. 
ie: ({message, data}) => (message, data)
My assumption is the output will look different now.

Also, a simple fix for now could be adding an Error as the known types for data. ie: object | Error | undefined. Then you could check if (data instanceof Error) and grab the properties needed, or just log the error as is.

And optionally, if we still wanted the data alongside the error you could do: 

```ts
(message: string, data?: object | Error, error?: Error)
```

Which would grant us some strictness with the typing, while also allowing some flexibility with logs with Errors + the data object metadata.